plutus-playground/plutus-playground-client/bower-packages.nix

```diff
@@ -0,0 +1,100 @@
+# Generated by bower2nix v3.2.0 (https://github.com/rvl/bower2nix)
```

Comment:
I'm confused - are we really supposed to have both a generated `bower-packages.nix` and a generated `yarn.nix`? Either way, can we add some documentation and ideally a checked-in script about how to regenerate these things.

Reply:
yes we are. javascript.

Reply:
@krisajenkins could you add something to the README